packages/muelu/research/mmayr/composite_to_regions/src/SetupRegionHierarchy_def.hpp

```diff
@@ -1195,6 +1195,48 @@ MakeCompositeDirectSolver(RCP<Xpetra::Matrix<Scalar, LocalOrdinal, GlobalOrdinal
   return coarseSolver;
 } // MakeCorseCompositeDirectSolver
 
+/* Create an AMG hierarchy for a composite operator
+ *
+ * Create the hierarchy object and perform the multigrid setup.
+ * Finally, the hierarhcy object will be ready to be applied during the region MG V-cycle call.
+ */
+template<class Scalar, class LocalOrdinal, class GlobalOrdinal, class Node>
+RCP<MueLu::Hierarchy<Scalar, LocalOrdinal, GlobalOrdinal, Node> >
+MakeCompositeAMGHierarchy(RCP<Xpetra::Matrix<Scalar, LocalOrdinal, GlobalOrdinal, Node> >& compOp, const std::string& xmlFileName)
+{
+#include "MueLu_UseShortNames.hpp"
+#include "Xpetra_UseShortNames.hpp"
```

Comment:
@mayrmt this is one is already included in "MueLu_Use_ShortNames.hpp"

Reply:
I'll remove it.